pricegraph/src/num.rs

```diff
@@ -77,6 +78,12 @@ pub fn is_strictly_positive_and_finite(value: f64) -> bool {
     value > 0.0 && value < f64::INFINITY
 }
 
+/// Returns true if an amount is considered a dust amount. See `MIN_AMOUNT`
+/// documentation for more details.
+pub fn is_dust_amount(amount: f64) -> bool {
+    amount < MIN_AMOUNT
+}
+
```

Comment:
The minimum amount is incredible low for most tokens. Do we know how effective this filtration will be for tokens with 18 decimal places? I mean it seems that even values like 0.00000000000001 ETH wouldn't be considered "dust orders"

Reply:
This is the `MIN_AMOUNT` as specified by the smart contract.

Reply:
Yes, but min amount is the WEI amount of a token, so the example above with such a little eth value is not considered a dust order.

Reply:
Pricegraph only uses amounts in WEI

Reply:
Perhaps there is some misunderstanding here. The goal is to exclude dust orders, but my claim is that what constitutes a dust order could be much larger than the smart contract's `MIN_AMOUNT`. For example, I would say that even `0.00001 ETH` is a dust order, but this is several factors of 10 larger than the `MIN_AMOUNT`.

Reply:
Ah I see. Maybe "dust order" is the incorrect name for this then. The purpose of excluding these orders is that matching them is incorrect (since the SC does not accept trades with these small amounts) so we have to make sure that the `pricegraph` doesn't find "ring" trades where one of the trades is tiny and too small to be considered by the SC.

Because of economic viability constraint, I would even say its OK to ignore orders of around `0.01 ETH` and below. But that would be a separate change.